dev/create-release/release-build.sh

```diff
@@ -235,40 +239,53 @@ if [[ "$1" == "package" ]]; then
       -DzincPort=$ZINC_PORT 2>&1 >  ../binary-release-$NAME.log
     cd ..
 
-    if [[ -n $R_FLAG ]]; then
-      echo "Copying and signing R source package"
-      R_DIST_NAME=SparkR_$SPARK_VERSION.tar.gz
-      cp spark-$SPARK_VERSION-bin-$NAME/R/$R_DIST_NAME .
+    R_DIST_NAME=SparkR_$SPARK_VERSION.tar.gz
+    cp spark-$SPARK_VERSION-bin-$NAME/R/$R_DIST_NAME .
```

Comment:
So, this `cp` operation is intentionally failing in our Jenkins, right?
- https://amplab.cs.berkeley.edu/jenkins/job/spark-master-package-sknapptest/6/console
```
cp: cannot stat `spark-3.0.0-SNAPSHOT-bin-without-hadoop-scala-2.12/R/SparkR_3.0.0-SNAPSHOT.tar.gz': No such file or directory
Skipping signing of R source package
cp: cannot stat `spark-3.0.0-SNAPSHOT-bin-without-hadoop-scala-2.12/python/dist/pyspark-3.0.0.dev0.tar.gz': No such file or directory
```

Reply:
hmm.  investigating.

Reply:
Yes. It's weird. It seems to be removed by `rm`.
```
+ tar czf spark-3.0.0-SNAPSHOT-bin-without-hadoop.tgz -C /home/jenkins/workspace/spark-master-package-sknapptest/spark-3.0.0-SNAPSHOT-bin-without-hadoop spark-3.0.0-SNAPSHOT-bin-without-hadoop
+ rm -rf /home/jenkins/workspace/spark-master-package-sknapptest/spark-3.0.0-SNAPSHOT-bin-without-hadoop/spark-3.0.0-SNAPSHOT-bin-without-hadoop
cp: cannot stat `spark-3.0.0-SNAPSHOT-bin-without-hadoop/R/SparkR_3.0.0-SNAPSHOT.tar.gz': No such file or directory
```

Reply:
yep...  definitely weird.

Reply:
the `rm -rf` is being called in `make-distribution.sh`...  gah, this is all so fragile.  and confusing.

Reply:
seriously, how does any of this stuff actually work?  ;)